src/main/java/com/uber/cadence/client/WorkflowStub.java

```diff
@@ -67,6 +67,10 @@
 
   WorkflowExecution start(Object... args);
 
+  CompletableFuture<WorkflowExecution> startAsync(Object... args);
+
+  CompletableFuture<WorkflowExecution> startAsyncWithTimeout(Long timeoutInMillis, Object... args);
```

Comment:
If you remove the method above, you can just call this startAsync

Reply:
It would be good to allow user just start without timeout, so will keep both APIs